dynamic-plugins/_utils/copy-plugins.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+const path = require('path');
+const process = require('process');
+
+process.chdir(path.join(__dirname, '..'));
+
+if (!process.argv[2]) {
+  console.log('Usage: node copy-plugins.js <destination directory>');
+  process.exit(1);
+}
+
+const destination = process.argv[2];
```

Comment:
if this arg is not set, does it default to current dir?

Reply:
No. The argument is required:

```
if (!process.argv[2]) {
  console.log('Usage: node copy-plugins.js <destination directory>');
  process.exit(1);
}
```